### PR TITLE
General - Fix command casing

### PR DESCRIPTION
**Pull request by BrettMayson**
**When merged this pull request will:**
- Changes command casing to match the wiki
- Used the in development `hemtt utils sqf case`, which I tested against ACE

### IMPORTANT

- If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

File: include/a3/ui_f/hpp/defineCommonGrids.inc
[the remark this answers is not in the document]
Comment (BrettMayson) [reply include/a3/ui_f/hpp/defineCommonGrids.inc:1]: It did them and I didn't commit the file, my initial thought was no but I'm fine either way. I don't think these files are likely to change much on the Arma side at this point.

[the remark this answers is not in the document]
Comment (PabstMirror) [reply include/a3/ui_f/hpp/defineCommonGrids.inc:1]: their macro changes are usually additive (e.g. they added the haze UI macros)
so it should be easy to fix any conflicts

**Comment by PabstMirror** [comment]
replaced by #10286 I think

**Comment by BrettMayson** [comment]
Oh, lol

**Pull request closed without merging**